### ISSUE TITLE
Linting hangs on media queries that can't be parsed

### ISSUE THREAD

**Issue by gdub22** [atfzl/eslint-plugin-css-modules#12]
I believe this is an issue with the underlying css parser (gonzales), but I think you are catching and ignoring anything it can't parse?

Linting hangs on any file containing this [postcss media query feature](https://github.com/postcss/postcss-media-minmax):
```css
@media (width > 300px) {
  body {
    font-size: 1.5rem;
  }
}
```

**Comment by atfzl** [comment]
`gonzales-pe` issue
